### PR TITLE
Updating Fantasy Land Compatibility

### PR DESCRIPTION
**Pull request by mwalkerwells**
Changed the test configuration (temporary), until this is resolved: https://github.com/fantasyland/fantasy-laws/pull/1

Bumped version to indicate the breaking change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,22 +21,25 @@
...
-    "grunt-contrib-jshint": "0.11.x",
-    "grunt-contrib-nodeunit": "0.4.x",
-    "fantasy-check": "0.1.6",
     "fantasy-helpers": "0.0.x",
-    "fantasy-identities": "0.x.x"
+    "fantasy-identities": "0.x.x",
+    "jsverify": "^0.8.2",
+    "mocha": "^3.5.0",
+    "nodeunit": "^0.11.1",
+    "sanctuary-type-classes": "^6.0.0"
```
Comment (davidchambers) [inline package.json]: Let's stick with the `"1.2.x"` form for consistency.

File: Gruntfile.js [hunk 2]
```diff
@@ -1,20 +0,0 @@
...
-                    'test/*.js'
-                ]
-            }
-        };
-
-    grunt.initConfig(config);
-    grunt.loadNpmTasks('grunt-contrib-jshint');
-    grunt.loadNpmTasks('grunt-contrib-nodeunit');
-    grunt.registerTask('default', ['jshint', 'nodeunit']);
-};
```
Comment (davidchambers) [inline Gruntfile.js]: I love to see gruntfiles removed. :)

**Comment by safareli** [comment]
If it was just me I would deprecate it in favor of fluture by @Avaq

**Comment by davidchambers** [comment]
I'm totally in favour of that option, @safareli.

**Comment by mwalkerwells** [comment]
Can we delete/hide this repo then?

**Comment by mwalkerwells** [comment]
Having stale projects, especially not up-to-spec, is both confusing & gives the impression fantasyland has lost momentum.

**Comment by safareli** [comment]
We can delete all code and add deprecation note in README

**Comment by davidchambers** [comment]
> We can delete all code and add deprecation note in README

#17

**Pull request closed without merging**